src/suggestions.ts

```diff
@@ -101,35 +138,40 @@ export function getSuggestedQuery(element, variant = 'get', method) {
 
   const placeholderText = element.getAttribute('placeholder')
   if (canSuggest('PlaceholderText', method, placeholderText)) {
-    return makeSuggestion('PlaceholderText', element, placeholderText, {
-      variant,
-    })
+    return makeSuggestion(
+      'PlaceholderText',
+      element,
+      placeholderText as string,
```

Comment:
I would rather let `makeSuggestion` handle `null`. Looks like we pass `null` a lot.  Otherwise this is unsound.

Reply:
The problem is we already check in the `canSuggest` function that the third parameter is not null but typescript not recognize it

Reply:
Would `placeholderText is string` help as the return value of `canSuggest`?